Makefile

```diff
@@ -42,51 +44,10 @@ ifndef IC_PROV_VPC_ID
 endif
 	cd acc_test && TF_ACC=1 go test -v -timeout 120m -count=1
 
-testtarget:
-ifndef TARGET
-	@echo "TARGET for test name must be set for running specific tests"
-	@exit 1
-endif
-
-ifeq (,$(findstring Test, $(TARGET)))
-	@echo "The target is not a test"
-	@exit 1
-endif
-
-ifneq (,$(findstring Acc, $(TARGET)))
-ifndef IC_USERNAME
-	@echo "IC_USERNAME for provisioning API must be set for acceptance tests"
-	@exit 1
-endif
-ifndef IC_API_KEY
-	@echo "IC_API_KEY for provisioning API must be set for acceptance tests"
-	@exit 1
-endif
-endif
-
-ifneq (,$(findstring Key, $TARGET))
-ifndef KMS_ARN
-	@echo "KMS_ARN must be set for encryption key tests"
-	@exit 1
-endif
-endif
-
-ifneq (,$(findstring CustomVPC, $TARGET))
-ifndef IC_PROV_ACC_NAME
-	@echo "IC_PROV_ACC_NAME must be set for custom VPC tests"
-	@exit 1
-endif
-ifndef IC_PROV_VPC_ID
-	@echo "IC_PROV_VPC_ID must be set for custom VPC tests"
-	@exit 1
-endif
-endif
-
-	cd test && TF_ACC=1 go test -v -run ${TARGET}
 
 install:
-	@if [ ! -d "$(HOME)/.terraform.d/plugins/" ]; then \
-		echo "$(HOME)/.terraform.d/plugins/ doesn't exist, creating..."; \
-		mkdir -p $(HOME)/.terraform.d/plugins/; \
+	@if [ ! -d $(INSTALL_FOLDER) ]; then \
+		echo "$(INSTALL_FOLDER) doesn't exist, creating..."; \
+		mkdir -p $(INSTALL_FOLDER); \
 	fi
-	cp ./bin/$(BIN_NAME)_$(VERSION) $(HOME)/.terraform.d/plugins/
+	cp ./bin/$(BIN_NAME)_v$(VERSION) $(INSTALL_FOLDER)
```

Comment:
does terraform with version <=0.12 recognise this folder?

Reply:
doubt it :P

Reply:
Does that mean we're just "dropping" support for local terraform <=0.12 development?
I think we had an offline discussion to add a new make goal to install for versions lower than 13 for that reason.

Reply:
As discussed in standup this morning I think it's ok to drop develpment support for lower versions

Reply:
I agree